libs/SalesforceSDK/src/com/salesforce/androidsdk/security/KeyStoreWrapper.java

```diff
@@ -227,33 +226,17 @@ private synchronized void createKeysIfNecessary(String algorithm, String name, i
                  * TODO: Remove this check once minVersion > 28.
                  */
                 if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
-                    try {
 
-                        /*
-                         * Disabling StrongBox for now, since it's too slow on Pixel 3
-                         * and Pixel 3 XL.
-                         *
-                         * TODO: Re-enable in Mobile SDK 7.2.
-                         */
-                        //keyGenParameterSpecBuilder.setIsStrongBoxBacked(true);
-                        keyGenParameterSpecBuilder.setIsStrongBoxBacked(false);
-                        kpg.initialize(keyGenParameterSpecBuilder.build());
-                        kpg.generateKeyPair();
-                    } catch (StrongBoxUnavailableException sb) {
```

Comment:
No need for the catch anymore?

Reply:
Nope - we don't use `StrongBox` anymore based on Google's recommendation.